### PR TITLE
[TH] Add "Ouendan", "Twitter" and update "Bancho (server)"

### PR DESCRIPTION
**Pull request by HelloYeew**
Add Thai translation for

- Ouendan
- Twitter

and update translation that outdated for

- Bancho (server)

### CODE REVIEW

File: wiki/Bancho_(server)/th.md [hunk 1]
```diff
@@ -1,14 +1,11 @@
...
-outdated: true
-outdated_since: 2a73fa15eba9ab6f0c0fbe547d71933628aedc64
+  - โครงสร้างพื้นฐาน
 ---
 
-# Bancho (เซิฟเวอร์แชท)
+# Bancho (เซิฟเวอร์เกม)
 
-*อย่าจำสลับกันกับ [BanchoBot](/wiki/BanchoBot)*
+*สำหรับการใช้งานอื่นๆ ดู [Bancho (คำคลุมเครือ)](/wiki/Disambiguation/Bancho)*
```
Comment (Trigonoculus) [inline wiki/Bancho_(server)/th.md:9]:
```suggestion
*สำหรับการใช้งานอื่น ๆ ดู [Bancho (คำคลุมเครือ)](/wiki/Disambiguation/Bancho)*
```

File: wiki/Bancho_(server)/th.md [hunk 2]
```diff
@@ -1,14 +1,11 @@
...
 
-**Bancho** เป็นเซิฟเวอร์ [Internet Relay Chat](/wiki/Internet_Relay_Chat) (*IRC*) สำหรับ osu! การลงชื่อเข้าใช้ในเกมด้วยบัญชี osu! จะเชื่อมต่อเข้ากับ Bancho โดยอัตโนมัติ
+**osu!bancho** เป็นเซิฟเวอร์ที่ใช้จัดการสถานะผู้ใช้, แชท และ บริการผู้เล่นหลายคน
```
Comment (Trigonoculus) [inline wiki/Bancho_(server)/th.md:11]:
```suggestion
**osu!bancho** คือเซิฟเวอร์ที่ใช้จัดการสถานะผู้ใช้, แชท และ บริการผู้เล่นหลายคน
```

File: wiki/Ouendan/th.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+stub: true
+tags:
+  - Osu! Tatake! Ouendan
+---
+
+# Ouendan
+
+Ouendan สามารถมีหลายความหมาย:
+
+- ในวัฒนธรรมญี่ปุ่นเป็นกลุ่มเชียร์คล้ายๆในวัฒนธรรมตะวันตกคือการให้กำลังใจทีมกีฬาโดยการใช้กลอง แตร ป้าย และเสียงเชียร์ <!-- I try not to change a meaning and make it easier to read.-->
```
Comment (Trigonoculus) [inline wiki/Ouendan/th.md:11]:
```suggestion
- ในวัฒนธรรมญี่ปุ่น เป็นกลุ่มเชียร์คล้าย ๆ ในวัฒนธรรมตะวันตก คือการให้กำลังใจทีมกีฬาโดยการใช้กลอง แตร ป้าย และเสียงเชียร์ <!-- I try not to change a meaning and make it easier to read.-->
```

**Review by Trigonoculus — CHANGES_REQUESTED** [review]
👍 suggest แก้นิดเดียวครับ

**Comment by HelloYeew** [comment]
Complete 👍

**Review by Trigonoculus — APPROVED** [review]
Looks good to me

**Review by C0NS03L — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**